pkg/ebpf/tracer.go

```diff
@@ -369,7 +369,19 @@ func (t *Tracer) GetActiveConnections(clientID string) (*network.Connections, er
 	conns := t.state.Connections(clientID, latestTime, latestConns, t.reverseDNS.GetDNSStats())
 	names := t.reverseDNS.Resolve(conns)
 
-	return &network.Connections{Conns: conns, DNS: names}, nil
+	kprobeStats := GetProbeTotals()
+	conntrackStats := t.conntracker.GetStats()
+	dnsStats := t.reverseDNS.GetStats()
+	stateTelemetry, _ := t.state.GetStats()["telemetry"].(map[string]int64)
+
+	tm := &network.ConnectionsTelemetry{
+		KprobesTriggered:    kprobeStats.hits,
+		KprobesMissed:       kprobeStats.miss,
+		ConntrackTotal:      conntrackStats["registers_total"] + conntrackStats["registers_dropped"],
```

Comment:
Let's do register and registers dropped separately since its of interest to know how many are dropped